src/transport/MixTransport.cpp

```diff
@@ -112,6 +113,33 @@ void MixTransport::getSpeciesFluxes(size_t ndim, const doublereal* const grad_T,
             sum[n] += fluxes[n*ldf + k];
         }
     }
+
+	//Modification: amibipolar diffusion. Ref.- Prager et al. 2007
+	//some dummy variables
+	doublereal* beta;
+	doublereal sum1;
+	doublereal sum2;
+	for (size_t n = 0; n < m_nDim; n++) {
+		for (size_t k = 0; k < m_nsp; k++){
+			sum1 = m_chargeSpecies[k] * m_chargeSpecies[k] * m_spwork[k] * m_molefracs[k];
```

Comment:
This loop does nothing. when it is finished, `sum1` is just the value for the last species. Do you mean `+=` instead of `=`? Same for `sum2`.